yellowbrick/model_selection/CVScores.py

```diff
@@ -0,0 +1,174 @@
+
+# coding: utf-8
+
+# In[ ]:
+
+# yellowbrick.model_selection.CVScores
+#
+#
+# Author:   Prema Damodaran Roman
+
+#
+# Copyright (C) 2018 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: CVScores.py [7f47800] pdamo24@gmail.com $
+
+##########################################################################
+## Imports
+##########################################################################
+
+import warnings
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+
+from yellowbrick.base import ModelVisualizer
+from sklearn.model_selection import cross_val_score
+
+##########################################################################
+## CVScores Visualizer
+##########################################################################
+
+class CVScores(ModelVisualizer):
+    """
+    CVScores displays cross validation scores as a bar chart and the 
+    average of the scores as a horizontal line
+
+    Parameters
+    ---------- 
+
+    model : a scikit-learn estimator
+            An object that implements ``fit`` and ``predict``, can be a
+            classifier, regressor, or clusterer so long as there is also a valid
+            associated scoring metric.
+            Note that the object is cloned for each validation.
+
+    ax : matplotlib.Axes object, optional
+            The axes object to plot the figure on.
+
+    cv : int, cross-validation generator or an iterable, optional
+            Determines the cross-validation splitting strategy.
+            Possible inputs for cv are:
+                - None, to use the default 3-fold cross-validation,
+                - integer, to specify the number of folds.
+                - An object to be used as a cross-validation generator.
+                - An iterable yielding train/test splits.
+
+            see the scikit-learn
+            `cross-validation guide <http://scikit-learn.org/stable/modules/cross_validation.html>`_
+            for more information on the possible strategies that can be used here.
+
+    scoring : string, callable or None, optional, default: None
+            A string or scorer callable object / function with signature
+            ``scorer(estimator, X, y)``. 
+    
+            See scikit-learn model evaluation documentation for names of possible metrics.
```

Comment:
As in the `cv` param; could we include a link to the scikit-learn documentation?